core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingRepository.scala

```diff
@@ -3,13 +3,20 @@ package org.broadinstitute.dsde.rawls.billing
 import org.broadinstitute.dsde.rawls.dataaccess.SlickDataSource
 import org.broadinstitute.dsde.rawls.model.{RawlsBillingProject, RawlsBillingProjectName}
 
+import java.util.UUID
 import scala.concurrent.Future
 
 /**
  * Data access for rawls billing projects
  */
 class BillingRepository(dataSource: SlickDataSource) {
 
+  def setBillingProfileId(projectName: RawlsBillingProjectName, billingProfileId: UUID): Future[Int] = {
```

Comment:
Should we maybe have a sub-package for DAOs?
I'm not adamant about it or anything, but it might make things feel a little more organized.

Reply:
Let's try it here for now, it's easy enough to move if need be.